server/interview/views.py

```diff
@@ -191,8 +195,111 @@ class NotifyInterview(APIView):
     def post(self, request):
         # TODO: Implement the notification service
         return Response({"detail": "Notification sent."})
+    
+class InterviewAll(APIView):
+    permission_classes = [IsAuthenticated]
+    # TODO: admin permission
+    def get(self, request):
+        interviews = Interview.objects.all()
+
+        # Check if there are no interviews
+        if not interviews:
+            return Response(
+                {"detail": "No interviews found."},
+                status=status.HTTP_404_NOT_FOUND
+            )
+        
+        # Prepare the response data
+        interview_data = []
+        for interview in interviews:
```

Comment:
It could but would just showing the id of the question. This is mainly for debugging after pairing so admin don't have to make extra call to api to get those information